File: 양승민/입문/src/lec01/lec01Main.java
package lec01;

import java.util.ArrayList;
import java.util.List;
import java.util.Optional;

public class lec01Main {
    public static void main(String[] args) {

        // 1. 변수 선언 키워드: var과 val의 차이점
        long number1 = 10L;
        final long number2 = 10L;

        // 2. Java에서의 기본형 타입
        Long number3 = 1_000L;

        // ++ Java에서의 컬렉션
        List<Integer> numbers = new ArrayList<>();

        // 3. Kotlin에서의 nullable 변수
        Optional<Long> number4 = null;

        // 4. Kotlin에서의 객체 인스턴스화
        Person person = new Person("양승민");

    }
}

File: 양승민/입문/src/lec01/lec01Main.kt
package lec01

fun main() {

    // 1. 변수 선언 키워드: var과 val의 차이점
    var number1: Long = 10L
    val number2: Long = 10L

    // 2. Kotlin에서의 기본형 타입
    var number3: Long = 1_000L

    // ++ Kotlin에서의 컬렉션
    val numbers = ArrayList<Int>()

    // 3. Kotlin에서의 nullable 변수
    var number4: Long? = null

    // 4. Kotlin에서의 객체 인스턴스화
    var person = Person("양승민")

}